changelog/6384.bugfix.rst

```diff
@@ -0,0 +1 @@
+Make --showlocals work also with --tb=short
```

Comment:
```suggestion
Make `--showlocals` work also with `--tb=short`.
```

Reply:
ok

Reply:
Missed the dot.. ;)
(please squash into a single commit then also with the next update - I'd await approval/feedback from others first)

Reply:
Right